README.md

```diff
@@ -84,19 +85,14 @@ export default class ReactView extends React.Component {
           "rowsSelector": "RPP",
         }}
         isDisabled={false}
-        value={{ value: 'a', label: 'The first letter in the alphabet, transformed with String.toLowerCase' }}
+        value={value}
```

Comment:
Does the new `value` contain a value and label separately as before or are they combined in this new version?

Reply:
`value` is a select object (change in react-select v1 => v2).